client/config/environment.js

```diff
@@ -37,17 +38,15 @@ module.exports = function(environment) {
     ENV.contentSecurityPolicy = {
       'default-src': "'none'",
       // add the domain here like analytics.google.com or data: or 'self'
-      'script-src':  "'self' 'unsafe-eval' 'https://www.google.com' ''",
+      'script-src':  "'self' 'unsafe-eval' 'unsafe-inline' www.google.com www.google-analytics.com",
```

Comment:
It seems like the script is not inline, you've put it in ga.js, so we shouldn't need 'unsafe-inline'  here, should we?